src/freerct.cpp

```diff
@@ -72,6 +75,21 @@ static void PrintUsage()
 	printf("\n");
 }
 
+/** Output command-line version info. */
+static void PrintVersion()
+{
+	printf("FreeRCT\n\n");
+
+	printf("Version               : %s\n",   _freerct_revision);
+	printf("Build ID              : %s\n",   _freerct_build_date);
+	printf("Installation directory: %s\n",   _freerct_install_prefix);
+	printf("User data directory   : %s\n\n", freerct_userdata_prefix());
+
+	printf("Homepage: https://github.com/FreeRCT/FreeRCT\n");
+
+	printf("\nFreeRCT is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2. FreeRCT is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with FreeRCT. If not, see <http://www.gnu.org/licenses/>\n");
```

Comment:
A few newlines in there make it simpler to read, and increases the chance all text is actually shown (some output devices may simply truncate whatever is right of the right-margin).

I know you can stick a newline anywhere in the string, but it's simpler if you do that at only one place (at the end). Easier to read, and easier to predict output from the code. If you ever need to search/replace stuff, using it at one spot makes automation easier as well.